### PR TITLE
Gas Gaussian Cloud: Add Missing Documentation

### PR DESCRIPTION
**Pull request by ax3l**
This adds the missing documentation in `gasConfig.param` for the GaussianCloud profile.

Was removed during refactoring since even the [original documentation](https://github.com/ComputationalRadiationPhysics/picongpu/blob/f31b31ab5afe8951ba58754fc6266a4ac07ffd20/src/picongpu/include/simulation_defines/param/gasConfig.param#L167-L173) was copy-pasted from an other profile (and therefore wrong).

### CODE REVIEW

File: src/picongpu/include/simulation_defines/param/gasConfig.param [hunk 1]
```diff
@@ -163,6 +167,7 @@ PMACC_STRUCT(GaussianCloudParam,
 /* definition of gas cloud profile*/
 typedef GaussianCloudImpl<GaussianCloudParam> GaussianCloud;
 
+
```
Comment (PrometheusPi) [inline src/picongpu/include/simulation_defines/param/gasConfig.param]: why is here a newline added?

Comment (ax3l) [inline src/picongpu/include/simulation_defines/param/gasConfig.param]: because all profiles are added by two newlines, this one was not.

Comment (ax3l) [inline src/picongpu/include/simulation_defines/param/gasConfig.param]: added more below now.

File: examples/Bunch/include/simulation_defines/param/gasConfig.param [hunk 2]
```diff
@@ -47,6 +47,13 @@ namespace gasProfiles
 {
 
 PMACC_STRUCT(GaussianCloudParam,
+    /** Gas Formula:
+     *     exponent = |globalCellPos - center| / sigma
+     *     density = e^[ gasFactor * exponent^gasPower ]
+     */
+    (PMACC_C_VALUE(float_X, gasFactor, -1.0))
```
Comment (PrometheusPi) [inline examples/Bunch/include/simulation_defines/param/gasConfig.param]: Could we use -1/2 as default value here as the standard [Gauss distribution](https://en.wikipedia.org/wiki/Gaussian_function)?

File: src/picongpu/include/simulation_defines/param/gasConfig.param [hunk 3]
```diff
@@ -138,6 +138,13 @@ typedef LinearExponentialImpl<LinearExponentialParam> LinearExponential;
 
 
 PMACC_STRUCT(GaussianCloudParam,
+    /** Gas Formula:
+     *     exponent = |globalCellPos - center| / sigma
+     *     density = e^[ gasFactor * exponent^gasPower ]
+     */
+    (PMACC_C_VALUE(float_X, gasFactor, 1.0))
+    (PMACC_C_VALUE(float_X, gasPower, 4.0))
```
Comment (PrometheusPi) [inline src/picongpu/include/simulation_defines/param/gasConfig.param]: same question here: why different than standard Gauss?

File: src/picongpu/include/simulation_defines/param/gasConfig.param (hunk 1 above)
Comment (PrometheusPi) [inline src/picongpu/include/simulation_defines/param/gasConfig.param]: :+1:

**Comment by PrometheusPi** [comment]
looks godd to me - just need a answer or  a fix to my two questions. 
@ax3l great job

File: examples/Bunch/include/simulation_defines/param/gasConfig.param (hunk 2 above)
Comment (ax3l) [inline examples/Bunch/include/simulation_defines/param/gasConfig.param]: I did not want to change your definition of your `Bunch` example (this line).
I would definitely change the default, good point.

Do you want to change it in the Bunch example as well?

File: src/picongpu/include/simulation_defines/param/gasConfig.param (hunk 3 above)
Comment (ax3l) [inline src/picongpu/include/simulation_defines/param/gasConfig.param]: oh yes, it's even wrong this way...

File: examples/Bunch/include/simulation_defines/param/gasConfig.param (hunk 2 above)
Comment (PrometheusPi) [inline examples/Bunch/include/simulation_defines/param/gasConfig.param]: Yes - change the Bunch example as well.

Comment (ax3l) [inline examples/Bunch/include/simulation_defines/param/gasConfig.param]: let's use `-0.5` and `2.0` for a real gauss

File: src/picongpu/include/simulation_defines/param/gasConfig.param (hunk 3 above)
Comment (ax3l) [inline src/picongpu/include/simulation_defines/param/gasConfig.param]: let's use `-0.5` and `2.0` for a real gauss

File: examples/Bunch/include/simulation_defines/param/gasConfig.param (hunk 2 above)
Comment (ax3l) [inline examples/Bunch/include/simulation_defines/param/gasConfig.param]: just the factor or also the power? (sry for dbl post)

**Comment by ax3l** [comment]
updated, thanks! :)

File: examples/Bunch/include/simulation_defines/param/gasConfig.param [hunk 4]
```diff
@@ -47,6 +47,13 @@ namespace gasProfiles
 {
 
 PMACC_STRUCT(GaussianCloudParam,
+    /** Gas Formula:
+     *     exponent = |globalCellPos - center| / sigma
+     *     density = e^[ gasFactor * exponent^gasPower ]
+     */
+    (PMACC_C_VALUE(float_X, gasFactor, -0.5))
```
Comment (ax3l) [inline examples/Bunch/include/simulation_defines/param/gasConfig.param]: changed :)

File: src/picongpu/include/simulation_defines/param/gasConfig.param [hunk 5]
```diff
@@ -138,6 +138,13 @@ typedef LinearExponentialImpl<LinearExponentialParam> LinearExponential;
 
 
 PMACC_STRUCT(GaussianCloudParam,
+    /** Gas Formula:
+     *     exponent = |globalCellPos - center| / sigma
+     *     density = e^[ gasFactor * exponent^gasPower ]
+     */
+    (PMACC_C_VALUE(float_X, gasFactor, -0.5))
```
Comment (ax3l) [inline src/picongpu/include/simulation_defines/param/gasConfig.param]: fixed :)

**Comment by ax3l** [comment]
@PrometheusPi can be merged :)

**Pull request merged**